locator/tablets.hh

```diff
@@ -229,6 +229,27 @@ struct resize_decision {
     sstring name() const;
 };
 
+struct table_load_stats {
+    uint64_t size_in_bytes = 0;
+    // Stores the minimum generation among all replicas, as coordinator wants to know if
+    // all replicas have completed splitting, which happens when they all store the
+    // generation of the current split decision.
+    api::timestamp_type split_ready_generation = api::max_timestamp;
```

Comment:
generation and timestamp have different units

Reply:
I disagree. what do you limit generation as a concept to its type? if a type can uniquely identify something, then it works as a generation.

Reply:
timestamp type has the two properties the satisfty the generation requirements

1) monotonically increasing globally.
2) can uniquely identify resize decision (as timestamp is taken by coordinator).

I don't have a sentiment for it though, I can switch to seq number if you want. But it seemed to me having a generation with typestamp type would convey information about when a resize started, by looking at system.tablets.

Reply:
@avikivity please take a look at my arguments above and from https://github.com/scylladb/scylladb/pull/16580#discussion_r1464977349, and tell me what you think.

Reply:
Or I can just call it resize_timestamp. If you don't have any objection, that's what I'll do in the next version.

Reply:
If it's consistent then okay. But beware that you're now introducing problems since the timestamp can go backwards if we move to a different node. With Raft we don't have to care about clock synchronization, and you're re-introducing the requirement.